v4/source/groups.yaml

```diff
@@ -1,3 +1,563 @@
+  '/groups':
+    get:
+      tags:
+        - groups
+      summary: Get groups
+      description: |
+        Retrieve the list of groups.
+
+        ##### Permissions
+        Must have `manage_system` permission.
+
+        __Minimum server version__: 5.11
+      parameters:
+        - name: page
+          in: query
+          description: The page to select.
+          default: "0"
+          type: string
+        - name: per_page
+          in: query
+          description: The number of groups per page.
+          default: "60"
+          type: string
```

Comment:
Other optional parameters include:
* `q` string to query groups by display name or name
* `include_member_count` boolean which adds the `member_count` attribute to each group JSON object
* `not_associated_to_team` string to exclude groups that are already linked to a given team via a `GroupTeam` table record.

Reply:
Thanks! Fixed.